app/javascript/elm/src/FixedSessionFilters.elm

```diff
@@ -13,19 +15,22 @@ import Ports
 type alias Model =
     { tags : Labels
     , profiles : Labels
+    , timeRange : TimeRange
     }
 
 
 defaultModel : Model
 defaultModel =
     { tags = Labels.empty
     , profiles = Labels.empty
+    , timeRange = TimeRange.defaultTimeRange
     }
 
 
 type alias Flags =
     { tags : List String
     , profiles : List String
+    , timeRange : Encode.Value
```

Comment:
Great that you started playing with decoders. As you said we should start stepping up the game and make the compiler protect us more with types